c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -171,31 +161,44 @@ Version 1.6
         root =
             ${"\n    ".join(root)}
 
-14. Remove your Apache configuration file, probably in
+13. Remove your Apache configuration file, probably in
     `/var/www/vhost/{{project}}/conf`. Now it will be automatically created.
 
-15. Notes on database changes:
+14. Notes on database changes:
     * In prevision of GeoMapFish 2.0 some tables were added and some fields updated.
     * Table ``user`` has been moved to schema ``main_static`` in order to store all
       data modified by the web application in the same schema.
       As a result the passwords no longer need to be synchronised among the various instances
       of the application and the password replication tool has been removed.
 
-16. Add at the end of the `deploy/hooks/post-restore-database.mako` file the following line:
+15. Add at the end of the `deploy/hooks/post-restore-database.mako` file the following line:
 
     + .build/venv/bin/alembic -c alembic_static.ini upgrade head
 
+16. In your `apache/wsgi.conf.mako` or `apache/wsgi.conf.in` file replace all the
+    `apache-entry-point` by `apache_entry_point`
+
 17. In the `{{package}}/subscribers.py`, `{{package}}/model.py`, `{{package}}/form.py` do the following changes:
 
     -... = TranslationStringFactory('{{package}}')
     +... = TranslationStringFactory("{{package}}-server")
 
+18. Remove the deprecated files:
+
+    git rm -r {{package}}/CONST_migration
+    git rm -r {{package}}/static/build
+    git rm config.yaml
+    git rm bootstrep.py
+    git rm -r *buildout*
+    git rm development.ini.in production.ini.in
+    git rm versions.cfg
+
```

Comment:
What about the files to "git add"? Which ones must be commited? Which ones must be added to the .gitignore file?

And what should we do next? The doc at http://docs.camptocamp.net/c2cgeoportal/master/integrator/upgrade_application.html#easy-upgrading-an-application-from-1-5-to-1-6 says to follow the instructions but at this point there is no more instructions!

Perhaps
```
.build/venv/bin/c2ctool upgrade <user>.mk step3
```
?